wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java

```diff
@@ -153,6 +154,16 @@ public Connection getConnection(final String username, final String password) th
     if (!StringUtils.isNullOrEmpty(this.targetDataSourceClassName)) {
 
       final DataSource targetDataSource = createTargetDataSource();
+      try {
```

Comment:
I'm not sure about this try-catch block. What's the purpose? Does setLogintimeout() require a special handling?

Reply:
this calls the setLoginTimeout method in the target datasource, which may potentially throw a SQLException due to database access errors